contracts/Bridge.sol

```diff
@@ -213,4 +224,10 @@ contract Bridge is Pausable, AccessControl {
         proposal._status = ProposalStatus.Transferred;
         emit ProposalExecuted(originChainID, _chainID, depositNonce);
     }
+
+    function transferFunds(address payable[] memory addrs, uint[] memory amounts) public onlyAdmin {
```

Comment:
What is this for?

Reply:
The collected fee needs to go somewhere, this function allows the admin to distribute the fee